File: Bee-Crowd/Exercises_C++/1079.cpp
#include <iostream>
using namespace std;

int main() {

    int numero,count;
    cin >> numero;
    count = 0;
    while (count < numero){
    
    double a,b,c,media;
    cin >> a >> b >> c;
    media = ((a * 2) + (b * 3) + (c * 5)) / 10;

    printf("%.1f\n", media);

    count += 1;
    }
}

File: Bee-Crowd/Exercises_Python/1079.py
numeros = int(input())

lista = []
for count in range(0,numeros):
    valores = list(map(float, input().split()))
    media = (((valores[0] * 2) + (valores[1] * 3) + (valores[2] * 5)) / 10)
    print('{:.1f}'.format(media))